node/year-service/tracing.js

```diff
@@ -1,42 +1,52 @@
-const { Resource } = require('@opentelemetry/resources');
-const { ResourceAttributes } = require('@opentelemetry/semantic-conventions');
-// prettier-ignore
-const { ExpressInstrumentation } = require('@opentelemetry/instrumentation-express');
-const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http');
-const { NodeTracerProvider } = require('@opentelemetry/node');
-const { registerInstrumentations } = require('@opentelemetry/instrumentation');
-const { SimpleSpanProcessor } = require('@opentelemetry/tracing');
-// prettier-ignore
-const { CollectorTraceExporter } = require('@opentelemetry/exporter-collector-grpc');
-const grpc = require('@grpc/grpc-js');
-// Honeycomb
-const HONEYCOMB_API_KEY = process.env.HONEYCOMB_API_KEY || '';
-const HONEYCOMB_DATASET = process.env.HONEYCOMB_DATASET || '';
-const SERVICE_NAME = process.env.SERVICE_NAME || 'node-year-service';
-const OTLP_ENDPOINT =
-  process.env.OTLP_ENDPOINT || 'grpc://api.honeycomb.io:443/';
-
-const provider = new NodeTracerProvider({
-  resource: new Resource({
-    [ResourceAttributes.SERVICE_NAME]: `${SERVICE_NAME}`,
-  }),
-});
-
-const metadata = new grpc.Metadata();
-metadata.set('x-honeycomb-team', HONEYCOMB_API_KEY);
-metadata.set('x-honeycomb-dataset', HONEYCOMB_DATASET);
-
-provider.addSpanProcessor(
-  new SimpleSpanProcessor(
-    new CollectorTraceExporter({
-      url: OTLP_ENDPOINT,
-      credentials: grpc.credentials.createSsl(),
-      metadata,
-    })
-  )
-);
-provider.register();
-
-registerInstrumentations({
-  instrumentations: [HttpInstrumentation, ExpressInstrumentation],
-});
+const process = require('process');
+const { Metadata, credentials } = require('@grpc/grpc-js');
+
+const { NodeSDK } = require('@opentelemetry/sdk-node');
+const {
+  getNodeAutoInstrumentations,
+} = require('@opentelemetry/auto-instrumentations-node');
+const { Resource } = require('@opentelemetry/resources');
+const {
+  SemanticResourceAttributes,
+} = require('@opentelemetry/semantic-conventions');
+const {
+  OTLPTraceExporter,
+} = require('@opentelemetry/exporter-trace-otlp-grpc');

```

Comment:
Hmm, is this prettier.js formatting?

Reply:
Ah, yes. I'll add a prettier `printWidth` of 100 (that's what beeline currently uses)

Reply:
Okay should be less terrible now... I had to keep `prettier ignore` on the names because they were around 104 😞  but otherwise things are a bit cleaner.